nxdrive/data/qml/EngineAccountItem.qml

```diff
@@ -0,0 +1,132 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import "icon-font/Icon.js" as MdiFont
+
+Rectangle {
+    id: control
+    width: parent ? parent.width : 0
+    height: authenticated ? 100 : 125
+    property bool authenticated: !api.has_invalid_credentials(uid)
+    property string forceUi: force_ui || wui
+    property bool sync_enabled: feat_synchronization.enabled
+
+    RowLayout {
+        anchors.fill: parent
+        anchors.rightMargin: 60
+
+        Rectangle {
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            border.width: 1
+            border.color: grayBorder
+            radius: 10
+
+            IconLabel {
+                id: disconnectIcon
+                icon: MdiFont.Icon.close
+                tooltip: qsTr("DISCONNECT") + tl.tr
+                iconColor: iconFailure
+                onClicked: {
+                    accountDeletion.open()
+                }
+                anchors.top: parent.top
+                anchors.right: parent.right
+                anchors.margins: 7
+            }
+
+            GridLayout {
+                columns: 2
+                columnSpacing: 50
+                rowSpacing: 10
+                anchors.fill: parent
+                anchors.margins: 14
+
+                // Server URL and Username
+                ScaledText { text: remote_user; color: label; font.bold: true }
+                Link {
+                    text: server_url
+                    onClicked: api.open_remote_server(uid)
+                }
+
+                // Server UI (Web-UI or JSF)
+                ScaledText {
+                    text: qsTr("SERVER_UI") + tl.tr
+                    color: label
+                }
+                RowLayout {
+                    id: uiSelect
+                    spacing: 5
+                    property string suffix: " (" + qsTr("SERVER_DEFAULT") + ")"
```

Comment:
I just saw this is not good.
Could you fix it at the same time? It should be:
```
qsTr("SERVER_DEFAULT" + tl.tr
```
And the translation should contain parenthesis.